Appraisals

```diff
@@ -0,0 +1,7 @@
+appraise "rails-4" do
+  gem "rails", "4.2.7"
```

Comment:
But do we need to be specific for the tests? We do not test rails security here. Are you suggesting changing this file every time there is a new security update?